src/components/AssignmentTexter/Controls.jsx

```diff
@@ -312,29 +344,27 @@ export class AssignmentTexterContactControls extends React.Component {
 
   handleMessageFormChange = ({ messageText }) => this.setState({ messageText });
 
-  handleOpenAnswerPopover = event => {
+  handleOpenAnswerResponsePopover = event => {
```

Comment:
So I understand this is combining handleOpenAnswerPopover and handleOpenResponsePopover, but I'm not seeing where the response parts were being called before so just unclear on the reasoning/how this changes things.

Reply:
Combining the two popovers' handlers like this should have no behavioral impact - just code simplification. Previously responsesPopover stuff was being handled [here](https://github.com/MoveOnOrg/Spoke/pull/1790/commits/20e6575c129b26b83be43d5d54cb626851acc9e8#diff-891594c99eab8026a02071b140273077L369). Now those state values are combined with the state values for the AnswerPopover in the same `setState()` call.

(Further code simplification is definitely still possible: e.g. we have `answerPopoverOpen ` and `responsePopoverOpen` but we only have one popover, whose parts can't be open independently.)